mautic/mautic
# Dont show empty fields in the contact and company detail view

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [no]
| New feature/enhancement? (use the a.x branch)      | [yes]
| Deprecations?                          | [no]
| BC breaks? (use the c.x branch)        | [no]
| Automated tests included?              | [no] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

If we view detailed information about a contact or company on the view page, all fields are displayed. We propose to display only filled fields in this PR

Before:
![image](https://user-images.githubusercontent.com/96085911/160633510-d8f9a202-773a-455b-be19-136c675142b4.png)

After:
![image](https://user-images.githubusercontent.com/96085911/160908710-07750788-d8b7-42fe-8ec4-f1d836f6978a.png)





#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new contact/company or open an existing one
3. Open details information
4. Only filled fields here




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11016"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
